.clang-format

```diff
@@ -3,24 +3,29 @@ Language:        Cpp
 # BasedOnStyle:  LLVM
 AccessModifierOffset: 0
 AlignAfterOpenBracket: DontAlign
+AlignConsecutiveMacros: false
```

Comment:
I'm tempted to turn this on since this is more like what the old style looked like.
But it doesn't seem to respect the column limit in all cases if I turn this on (`s_longlines` complained).

Maybe we can just get rid of that script though. It is enforced (kind of) by Clang Format and in this single case it only overshot by two characters.